src/Zoom.js

```diff
@@ -5,14 +5,27 @@ import { fetchImage, getMaxDimensionScale, getScale } from './helpers'
 
 import Overlay from './Overlay'
 
+/**
+ * The `tmpSrc` and `TmpImg` code here is a workaround
+ * for a longstanding Firefox issue where replacing the
+ * `src` of an `<img>` tag requires calculations and
+ * results in a "flicker" of sorts.
+ *
+ * See https://github.com/rpearce/react-medium-image-zoom/issues/96
+ * for further details.
+ */
+
+const isFirefox = typeof InstallTrigger !== 'undefined'
```

Comment:
Seems [InstallTrigger](https://developer.mozilla.org/en-US/docs/Web/API/InstallTrigger) is not on standard track, which means it might be dropped by Firefox in the future?